apps/site/lib/site_web/controllers/customer_support_controller.ex

```diff
@@ -295,10 +302,71 @@ defmodule SiteWeb.CustomerSupportController do
       service: params["service"],
       subject: params["subject"],
       no_request_response: params["no_request_response"] == "on",
-      incident_date_time: params["date_time"]
+      incident_date_time: params["date_time"],
+      mode: params["mode"],
+      line: params["route"],
+      vehicle: params["vehicle"]
     })
   end
 
+  @spec get_all_modes() :: list
+  def get_all_modes() do
+    # get options for subway, commuter rail and ferry:
+    subway_cr_ferry_opts =
+      Enum.map([1, 2, 4], fn mode ->
+        mode_name = SiteWeb.ViewHelpers.mode_name(mode)
+        [key: mode_name, value: mode_name]
+      end)
+      |> List.insert_at(0, key: "Select", value: " ")
+
+    # special case for bus:
+    opts_with_bus =
```

Comment:
I created that special case because the ticket said "…with the one exception that we should use "Bus Other” instead of “Bus”, per request from Sasha." I might be able to come up with some shortcut.

Reply:
I haven't looked at this code in details, so maybe this is already happening, but I just wanted to clarify that we should show "Bus" to the user, and then submit "Bus Other" as the value to Heat. Sorry, I didn't describe that very clearly in the ticket.

Reply:
Yes, that is how I understood it (and how the `select` looks like):<br/>
<img width="390" alt="image" src="https://user-images.githubusercontent.com/61979382/95994709-9bb33780-0dfe-11eb-9174-d3e7359281d8.png">

Reply:
Thanks for the explanation!